mobile/src/Student/screens/StudentGameIntro/IntroInfo/index.js

```diff
@@ -4,15 +4,18 @@ import { scale, verticalScale } from "react-native-size-matters"
 import { fontFamilies, fonts, colors, fontWeights } from '../../../../utils/theme'
 const win = Dimensions.get('window');
 
-const IntroInfo = ({ monsterStyles, Monsters, phone, sticker, text, showSticker }) => {
+const IntroInfo = ({ monsterStyles, monsters, phone, sticker, text, showSticker }) => {
+    if ({ monsterStyles }.length != 2) {
+        console.debug("'monsterStyles' must be an array of 2 entries")
```

Comment:
Since this can cause the app crashes, for future, probably better to use `console.error` and also return an empty `View` otherwise this still causes the app crashes in stores.

nit: You can omit `{}` around `monsterStyles` and I believe you meant checking both `monsterStyles` and `monsters`

Reply:
Thank you!